Makefile

```diff
@@ -1,6 +1,8 @@
 #---------------------------------------------------------------------------------
 .SUFFIXES:
 #---------------------------------------------------------------------------------
+export DEVKITPRO=/opt/devkitpro #Netbeans on Linux only. Comment if not the same directory/unneeded.
```

Comment:
This shouldn't be included in the repository (I believe Netbeans supports cmake well, you could try to use that instead)